stream/src/main/scala/org/apache/pekko/stream/javadsl/Flow.scala

```diff
@@ -1894,6 +1894,72 @@ final class Flow[In, Out, Mat](delegate: scaladsl.Flow[In, Out, Mat]) extends Gr
       case elem if clazz.isInstance(elem) => creator.create()
     }
 
+  /**
+   * Recover allows to send last element on failure and gracefully complete the stream
```

Comment:
Could you add a full stop at the end of this line? And also in other places where this same comment is added in this PR.